htmlutil/htmlutil.go

```diff
@@ -156,3 +156,121 @@ func GetText(n *html.Node) string {
 	}
 	return strings.Trim(buffer.String(), "\n")
 }
+
+// GetApproximateTextWidth returns the approximate width of the given text for the given font size (in pixels), assuming a sans-serif font.
+func GetApproximateTextWidth(text string, fontSize int) float64 {
+	size := 0.0
+	fSize := float64(fontSize)
+	if fontSize == 0 {
+		fSize = 14.0 // default font size on ons site
+	}
+	spacing := 0.0286 * fSize // allow for some spacing between letters
```

Comment:
I would consider making this a constant with a comment to explain what it means - its a very specific number but looking at it I have no clue what it means in this context